test/golden/oneType.json

```diff
@@ -25,6 +25,14 @@
         "bool": true,
         "unit": []
     },
+    "myUnit": {
+        "tag": "MyUnit",
+        "contents": []
+    },
+    "myResult": {
+        "tag": "Err",
+        "contents": "clashing test"
+    },
```

Comment:
How do you generate pretty-printed content of this file? I wonder, what should I do when I change `OneType`  definition to make tests pass...

Reply:
Oh, this edit was made manually, so I think that we need a strategy to deal with such things... Do you have anything in mind?

Reply:
We can use `aeson-pretty` package to create this file from `OneType` automatically. Probably as a part of `generate-elm`, but not sure...

* http://hackage.haskell.org/package/aeson-pretty

But this is definitely a separate issue